recipe/recipe.yaml

```diff
@@ -21,31 +21,37 @@ build:
       then:
         # Unix recipe
         - export OPENSSL_DIR="$PREFIX"
-        - cargo install --locked --bin rattler-build --root ${PREFIX} --path . --no-track
+        - export CARGO_PROFILE_RELEASE_LTO=true
+        - export CARGO_PROFILE_RELEASE_STRIP=symbols
```

Comment:
this can be done as environment variables instead, see https://conda-forge.org/docs/maintainer/example_recipes/rust/